framework-crates/objc2-virtualization/translation-config.toml

```diff
@@ -0,0 +1,16 @@
+framework = "Virtualization"
+crate = "objc2-virtualization"
+required-dependencies = ["objc2-app-kit", "objc2-foundation"]
```

Comment:
From what I can tell, a lot of the functionality does not require AppKit, so please leave that out the required list.

See [this doc](https://docs.rs/objc2/0.5.1/objc2/topics/about_generated/cargo_features/index.html) for more information on dependency features.

Reply:
Thanks. Removed from `requred-dependencies`.